control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -196,7 +196,7 @@ func ReconcileKubeAPIServerDeployment(deployment *appsv1.Deployment,
 			// The KAS takes 90 seconds to finish its graceful shutdown, give it enough
 			// time to do that + 5 seconds margin. The shutdown sequence is described
 			// in detail here: https://github.com/openshift/installer/blob/master/docs/dev/kube-apiserver-health-check.md
-			TerminationGracePeriodSeconds: ptr.To[int64](95),
+			TerminationGracePeriodSeconds: ptr.To[int64](130),
```

Comment:
Please set to 135, mainly for parity with standalone, see
https://github.com/openshift/cluster-kube-apiserver-operator/blob/4dce19de399c07cf34f6e1f4aba520aa74ab3304/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L495-L499

Also, update the comment above this line to reflect new reality.